lib/handlebars/runtime.js

```diff
@@ -85,6 +85,23 @@ export function template(templateSpec, env) {
       return typeof current === 'function' ? current.call(context) : current;
     },
 
+    splat: function() {
+      return Utils.extend.apply(null, arguments);
```

Comment:
Just saw [this](https://github.com/wycats/handlebars.js/pull/1149/files#diff-12fc6be51b9642b813f72c8bd16d891aR528) so please disregard :) Do we have test coverage asserting that the first object is not mutated?